### ISSUE TITLE
Proposal to cease building jdk8 on zLinux

### ISSUE THREAD

**Issue by adamfarley** [adoptium/temurin-build#3381]
**Proposition**

We don't release JDK8 on zLinux, so we should stop building it and testing it.

**Purpose of Issue**

To outline the proposal, and give people a chance to discuss/object.

**Timescale**

I'm proposing to make this change at end-of-day on the 12th of June (barring objections/ongoing debate), to give everyone a chance to be heard.

**Proposal Pros**
- Makes build triage faster (fewer builds).
- Saves space.
- Saves time fixing the jdk8 zLinux builds/tests when they break ([example: test timeout](https://ci.adoptium.net/job/Test_openjdk8_hs_sanity.openjdk_s390x_linux/812/console)).
- Saves machine time.

**Proposal Cons**
- No more jdk8 nightly builds on zLinux. (Only a con if anyone actually uses these).
- Without nightly builds, any issues preventing jdk8 zLinux builds from completing (when/if we need them in the future) will accumulate, undiscovered until we try building these again.

**Comment by sxa** [comment]
I'd probably prefer to keep it in as "untested" for anyone that wants it (since the issue that lead to this was the sanity.openjdk tests hitting a 10 hour timeout) but equally I wouldn't mourn its loss if it was stopped completely.

I kicked off some sanity.openjdk runs that ran on various machines yesterday and it was taking up to 12 hours:

For reference jobs 811 through 815 in this graphic  ran on ubuntu2204-1, docker-sles12-1, docker-sles12-1, rhel8-2, and ubuntu2004-1 respectively. The last three yesteday had the timeout increased so it didn't stop at 10 hours.

![image](https://github.com/adoptium/temurin-build/assets/6487691/6580080d-fb95-4ff0-b06d-9d6edb28c30f)

**Comment by smlambert** [comment]
As per June 7th meetings, and will be minuted as part of the PMC meeting, we will stop building jdk8 zLinux (Hotspot), and if people require a JDK8 build for this platform, direct them to Semeru.

**Comment by adamfarley** [comment]
As per the comments above, my current plan is to create the PR to cease JDK8 zLinux building and testing on June 12th.

It will be created in the morning, I'll update the community in #build and #testing-aqavit to ensure everyone has the opportunity to come back from any holidays and object.

Barring objection, the PR will be merged around 5pm BST (12pm EST) that same day.

TLDR: This weekend will be the last nightly JDK8 zLinux build.

**Comment by adamfarley** [comment]
Gave it an extra day for any vacationers to sort through their emails.

PR has been created [here](https://github.com/adoptium/ci-jenkins-pipelines/pull/725)

**Comment by adamfarley** [comment]
No further responses. Merging now.
